GVRf/Framework/jni/shaders/material/unlit_fbo_shader.cpp

```diff
@@ -0,0 +1,125 @@
+/* Copyright 2015 Samsung Electronics Co., LTD
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "unlit_fbo_shader.h"
+
+#include "gl/gl_program.h"
+#include "objects/material.h"
+#include "objects/mesh.h"
+#include "objects/components/render_data.h"
+#include "objects/textures/texture.h"
+#include "util/gvr_gl.h"
+
+namespace gvr {
+static const char VERTEX_SHADER[] = "attribute vec4 a_position;\n"
+        "attribute vec4 a_tex_coord;\n"
+        "uniform mat4 u_mvp;\n"
+        "varying vec2 v_tex_coord;\n"
+        "void main() {\n"
+        "  v_tex_coord.x = a_tex_coord.x;\n"
+        "  v_tex_coord.y = 1.0f - a_tex_coord.y;\n"
```

Comment:
Not as important as fragment shader opacity calculation (fewer vertex shader invocations compared for fragment shader invocations) -- but why not pass in the required y tex coord directly? Why do you need to subtract the y coord from 1?